components/js/ops-overview.js

```diff
@@ -1,470 +1,557 @@
+    row_container_id: "panel-row-2",
+    display_name: "Signal Work Orders",
+    icon: "wrench",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    caption: "Work orders completed at traffic signal assets",
+    query:
+      "select count(created_date) as count where upper(work_order_status) in ('SUBMITTED', 'CLOSED') and fiscal_year=" +
+      fiscal_year,
+    resource_id: "v2ig-5yw3",
+    data_transform: function(x) {
+      return [x[0]["count"]];
+    },
+    update_event: "knack_data_pub_work_orders_signals_knack_socrata"
+  },
+  {
+    id: "bcycle-trips",
+    row_container_id: "panel-row-2",
+    display_name: "  B-Cycle Trips",
+    icon: "bicycle",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    resource_id: "cwi3-ckqi",
+    caption: "# of Austin B-Cycle trips taken this year.",
+    query: (function() {
+      var y = new Date().getFullYear().toString();
+      return (
+        "select count(checkout_date) as count where date_extract_y(checkout_date)=" +
+        y
+      );
+    })(),
+    data_transform: function(x) {
+      return [x[0]["count"]];
+    },
+    update_event: "bcycle_trip_pub"
+  },
+  {
+    id: "dockless-trips-total-count",
+    row_container_id: "panel-row-4",
+    display_name: "  Total Trips (scooter & bike)",
+    icon: "mobile",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    resource_id: "7d8e-dm7r",
+    caption: "# of total Dockless Mobility trips taken.",
+    query: (function() {
+      return "select count(id) as total_trips where trip_distance * 0.000621371 >= 0.1 and trip_distance * 0.000621371 < 500 and trip_duration < 86400";
+    })(),
+    data_transform: function(x) {
+      return [x[0]["total_trips"]];
+    },
+    update_event: "dockless_trips"
+  },
+  {
+    id: "dockless-trips-total-distance",
+    row_container_id: "panel-row-4",
+    display_name: "Total Distance (Miles)",
+    icon: "tachometer",
+    init_val: 0,
+    format: "thousands",
+    infoStat: true,
+    resource_id: "7d8e-dm7r",
+    caption: "Total miles from Dockless Mobility trips.",
+    query: (function() {
+      return "select sum(trip_distance) * 0.000621371 as total_miles where trip_distance * 0.000621371 >= 0.1 and trip_distance * 0.000621371 < 500 and trip_duration < 86400";
+    })(),
+    data_transform: function(x) {
+      return [x[0]["total_miles"]];
+    },
+    update_event: "dockless_trips"
+  },
+  {
+    id: "dockless-trips-avg-distance",
+    row_container_id: "panel-row-4",
+    display_name: " Average Trip Distance (Miles)",
+    icon: "tachometer",
+    init_val: 0,
+    format: "decimal",
+    infoStat: true,
+    resource_id: "7d8e-dm7r",
+    caption: "Average distance of all Dockless Mobility trips (miles)",
+    query: (function() {
+      return "select avg(trip_distance) * 0.000621371 as avg_miles where trip_distance * 0.000621371 >= 0.1 and trip_distance * 0.000621371 < 500 and trip_duration < 86400";
```

Comment:
Should we include a note or caveat that we filter for trips greater than .1 mile and less than 500? Also why 500? Seems like even 50 would be better based on the expected mileage capacity for batteries.